super-mode-calculator/src/lambdas/query/regionSql.ts

```diff
@@ -20,3 +20,49 @@ ELSE
 END
 AS region
 `;
+
+export const REGION_SQL_v2 = `
```

Comment:
what's the difference here?

Reply:
It's the column names in different tables-for region datalake.fact_acquisition_event   has country_Code whereas  online_traffic.fact_page_view_anonymised has country_key.The current super mode has countryCode from the Athena table in REGION_SQL

Reply:
Ok that makes sense.
It would be helpful if the names could make that clear (instead of v2/v3)
Or perhaps a function? e.g.
`const regionSql = (columnName: string): string => ...`